ci-operator/config/openshift/hypershift/openshift-hypershift-release-4.15__mce.yaml

```diff
@@ -25,9 +45,16 @@ promotion:
     namespace: stolostron
 releases:
   latest:
-    integration:
```

Comment:
I think this results in the `publish` postsubmit job being built on top of the promoted nightly instead of the integration stream.  Thus the postsubmit job will not include the change the triggered it.  This comment applies to all changed __mce.yaml files.